browser/src/robocorp/browser/__init__.py

```diff
@@ -277,21 +302,147 @@ def screenshot(
 
 
 def install(browser_engine: BrowserEngine, force: bool = False):
-    """
-    Downloads and installs the given browser engine.
-
-    Note: Google Chrome or Microsoft Edge installations will be installed
-    at the default global location of your operating system overriding your
-    current browser installation.
+    """Downloads and installs the given browser engine.
 
     Args:
-        browser_engine: Browser engine which should be installed
+        browser_engine: Browser engine which should be installed.
+        force: Force reinstall of stable browser channels.
+
+    Note:
+        Google Chrome or Microsoft Edge installations will be installed
+        at the default global location of your operating system overriding your
+        current browser installation.
     """
     from . import _engines
 
     _engines.install_browser(browser_engine, force=force)
 
 
+class BrowserConfig:
+    """Browser config description here.
+
+    Put what you want in it. Testing.
+    """
+    def __init__(
+        self,
+        browser_engine: BrowserEngine = BrowserEngine.CHROMIUM,
+        install: Optional[bool] = None,
+        headless: Optional[bool] = None,
+        slowmo: int = 0,
+        screenshot: str = "only-on-failure",
+        isolated: bool = False,
+        skip_playwright_stop: bool = False,
+    ):
+        """Configure browser engire class.
+        
+        Args:
+            browser_engine: Browser engine which should be used
+                default="chromium"
+                choices=["chromium", "chrome", "chrome-beta", "msedge",
+                         "msedge-beta", "msedge-dev", "firefox", "webkit"]
```

Comment:
As stated above, what would be the rendering of this and are we fine with the final formatting?

As [this](https://9b5187a6.documentation-5av.pages.dev/python/robocorp/robocorp-browser/api#configure) confuses me a little bit:
<img width="712" alt="Screenshot 2024-03-26 at 1 14 01 PM" src="https://github.com/robocorp/robocorp/assets/709053/42d4214d-14b1-40ef-bb52-4a3a4df09d4a">

We can go with something like (as the rendered Markdown):
> Browser engine which should be used. Choices: `"chromium"`, `"chrome"`, ... (defaults to `"chromium"`)